client/app/telecom/telephony/alias/configuration/feature/contactCenterSolution/statistics/telecom-telephony-alias-configuration-statistics.html

```diff
@@ -0,0 +1,117 @@
+                <dd data-translate="telephony_alias_configuration_stats_qos_detail"
+                    data-translate-values="{ 'percentage': $ctrl.getQoS() }"></dd>
+            </div>
+            <dt data-translate="telephony_alias_configuration_statistics_calls_total_received"></dt>
+            <dd data-ng-bind="$ctrl.stats.callsTotal"></dd>
+            <dt data-translate="telephony_alias_configuration_statistics_calls_answered"></dt>
+            <dd data-ng-bind="$ctrl.stats.callsAnswered"></dd>
+            <dt data-translate="telephony_alias_configuration_statistics_calls_lost"></dt>
+            <dd data-ng-bind="$ctrl.stats.callsLost"></dd>
+            <dt data-translate="telephony_alias_configuration_statistics_calls_average_wait_time"></dt>
+            <dd data-ng-bind="$ctrl.getAverageWaitTime() | tucDuration"></dd>
+            <dt data-translate="telephony_alias_configuration_statistics_calls_average_duration"></dt>
+            <dd data-ng-bind="$ctrl.getAverageCallTime() | tucDuration"></dd>
+        </dl>
+
+        <h3 class="oui-heading_underline"
+            data-translate="telephony_alias_configuration_statistics_calls_pending_title"></h3>
+        <dl>
+            <dt data-translate="telephony_alias_configuration_statistics_calls_pending_ongoing"></dt>
+            <dd data-ng-bind="$ctrl.getOngoingCalls().length"></dd>
+            <dt data-translate="telephony_alias_configuration_statistics_calls_pending_waiting"></dt>
+            <dd data-ng-bind="$ctrl.getPendingCalls().length"></dd>
+            <dt data-translate="telephony_alias_configuration_statistics_calls_pending_maxWait"></dt>
+            <dd>
+                <tuc-elapsed-time data-ng-if="$ctrl.getMaxWaitTime()"
+                                  data-from="$ctrl.getMaxWaitTime()"></tuc-elapsed-time>
+                <span data-ng-if="!$ctrl.getMaxWaitTime()">-</span>
+            </dd>
+        </dl>
+
+        <oui-datagrid class="my-5"
+                      data-rows="$ctrl.calls">
+            <oui-column data-title="'telephony_alias_configuration_statistics_calls_calling_number' | translate"
+                        data-property="callerIdNumber">
+                <span class="text-capitalize"
+                      data-ng-bind="$row.callerIdNumber | tucPhoneNumber"></span>
+            </oui-column>
+            <oui-column data-title="'telephony_alias_configuration_statistics_calls_queue_time' | translate"
+                        data-property="begin">
+                <tuc-elapsed-time data-from="$row.begin"></tuc-elapsed-time>
+            </oui-column>
+            <oui-column data-title="'telephony_alias_configuration_statistics_calls_time' | translate"
+                        data-property="answered">
+                <span class="text-capitalize"
+                      data-ng-if="!$row.answered"
+                      data-ng-bind="$row.answered ? 'yes' : 'no' | translate"></span>
```

Comment:
the `ng-bind` will always evaluate to the `no` translation when displayed

Reply:
Also, do you have a `yes` and `no` translation ids ?

Reply:
@FredericEspiau  For sure => https://github.com/ovh-ux/ovh-manager-telecom/blob/master/client/app/common/translations/Messages_fr_FR.xml#L72